.gitignore

```diff
@@ -5,3 +5,6 @@
 *.html
 urllog.txt
 *log
+.DS_Store
+tox.ini
```

Comment:
Why tox.ini? Generally it defines the rules for running tests and static code analysis. I don't think it should be in this list.